src/js/components/TaskMesosUrlComponent.jsx

```diff
@@ -0,0 +1,65 @@
+var React = require("react/addons");
+
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var InfoStore = require("../stores/InfoStore");
+
+var TaskMesosUrlComponent = React.createClass({
+  displayName: "TaskMesosUrlComponent",
+  propTypes: {
+    className: React.PropTypes.string,
+    task: React.PropTypes.object.isRequired,
+    text: React.PropTypes.string
+  },
+
+ getInitialState: function () {
+    return {
+      info: InfoStore.info
+    };
+  },
+
+  componentDidMount: function () {
+    InfoActions.requestInfo();
+  },
+
+  componentWillMount: function () {
+    InfoStore.on(InfoEvents.CHANGE, this.onInfoChange);
+  },
+
+  onInfoChange: function () {
+    this.setState({
+      info: InfoStore.info
+    });
+  },
+
+  buildUrl: function (task, info) {
```

Comment:
In my opinion, this component is a good option, because now it is used in two other components 
( `TaskDetailComponent` and `TaskListItemComponent`). 

In the near future I would like to create a tab where we will be see the last failure task of app, in which a component `TaskMesosUrlComponent` will be used.